### PR TITLE
[536925] prevent npe in logging

### PR DESCRIPTION
**Pull request by cdietrich**
[536925] prevent npe in logging
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=536925

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=536925

File: org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/hyperlinking/SingleHoverShowingHyperlinkPresenter.xtend [hunk 1]
```diff
@@ -79,7 +79,7 @@ class SingleHoverShowingHyperlinkPresenter implements InvocationHandler {
 				if (it !== null && it.hyperlinkRegion !== null) {
 					list.add(it)
 				} else {
-					log.warn('Filtered invalid hyperlink: '+ it.getClass.name)
+					log.warn('Filtered invalid hyperlink: '+ it?.class?.name)
```
Comment (szarnekow) [inline org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/hyperlinking/SingleHoverShowingHyperlinkPresenter.xtend]: Why not just a logging of the value of it?

Comment (cdietrich) [reply org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/hyperlinking/SingleHoverShowingHyperlinkPresenter.xtend]: cant tell why its done this way.

**Pull request merged**